frontend/src/styles/HetComponents/HetQuoteLink.tsx

```diff
@@ -0,0 +1,18 @@
+import type React from 'react'
+import { FormatQuote } from '@mui/icons-material'
+
+interface HetQuoteLinkProps {
+  href: string
+}
+
+const HetQuoteLink: React.FC<HetQuoteLinkProps> = ({ href }) => {
+  return (
+    <a href={href}>
+      <span>
+        [<FormatQuote className='text-text' />]
```

Comment:
same here, I think the icon needs an accessible label like "open link in a new window".